content/english/python-basics/data-types/numbers.md

```diff
@@ -53,7 +58,7 @@ Using `str(5)` turns the number into the string `"5"`. Converting one type of th
 
 Here is how to print a random number between 1 (inclusive) and 10 (inclusive):
 
-<font color="blue">from</font> random <font color="blue">import</font> randint
+<font color="blue">from</font> random <font color="blue">import</font> randint  <font color="green">#This line imports the library needed</font>
```

Comment:
Maybe just replace with a code block (remove the slashes at the beginning)

```
 \```python 
 # This line imports the library needed
 from random import randint
 \```
```